es/i18n/es.json

```diff
@@ -397,9 +397,15 @@
     "errors": {
         "registration-check-token": "Error de registro de usuario. Por favor verifique su token e inténtelo nuevamente.",
         "not-registered-cant-contact": "El usuario no está registrado, por lo que no se puede contactar con el servidor.",
+        "while-initializing-label": "While initializing Label tab: ",
+        "while-loading-pipeline-range": "Error while loading pipeline range",
```

Comment:
These have to be the spanish translations, right?

Reply:
updates made in [ef9628e](https://github.com/e-mission/e-mission-translate/pull/52/commits/ef9628e23bacd9dbdf37a12c0867aa187a62fd86)